packages/expo-constants/src/ExponentConstants.web.ts

```diff
@@ -31,29 +29,29 @@ export default {
   get linkingUri(): string {
     return location.origin + location.pathname;
   },
-  get expoRuntimeVersion(): ?string {
+  get expoRuntimeVersion(): string | null {
     console.warn(`ExponentConstants.expoRuntimeVersion: is unimplemented on this platform.`);
     return null;
   },
-  get deviceName(): ?string {
+  get deviceName(): string | null {
     return null;
   },
-  get systemFonts(): ?Array<string> {
+  get systemFonts(): Array<string> {
     return [];
   },
   get statusBarHeight(): number {
     return 0;
   },
-  get deviceYearClass(): ?string {
+  get deviceYearClass(): string | null {
     console.warn(`ExponentConstants.deviceYearClass: is unimplemented on this platform.`);
     return null;
   },
-  get manifest(): { [string]: any } {
+  get manifest(): { [manifestKey: string]: any } {
     /* TODO: Bacon: Populate */
 
     return {};
   },
-  getWebViewUserAgentAsync(): Promise {
-    return navigator.userAgent || null;
+  getWebViewUserAgentAsync(): Promise<string | null> {
```

Comment:
```suggestion
  async getWebViewUserAgentAsync(): Promise<string> {
    return navigator.userAgent;
  }
```

instead of creating the Promise manually, also don't think we need `|| null`, it is always a string here https://html.spec.whatwg.org/multipage/system-state.html#client-identification